Tuist/Interfaces/SwiftUI/Project/Podfile

```diff
@@ -14,7 +14,8 @@ target '{PROJECT_NAME}' do
   pod 'Kingfisher'
 
   # Backend
-  pod 'Alamofire'
+  pod 'Moya/Combine'
```

Comment:
We need to close this issue https://github.com/nimblehq/ios-templates/issues/324 first, before adding Moya to other template

Reply:
solved in [6346fc6](https://github.com/nimblehq/ios-templates/pull/506/commits/6346fc696cbeea7b6a6717717e60db4a66203597)